rawpointer_test.go

```diff
@@ -232,3 +232,24 @@ func TestRawPointerTotalListSize(t *testing.T) {
 		}
 	}
 }
+
+func TestLandingPadNearPointer(t *testing.T) {
+	tests := []struct {
+		far     rawPointer
+		tag     rawPointer
+		landing rawPointer
+	}{
+		{0x08, 0x2000200000000, 0x2000200000000},
+		{0xa0, 0x2000200000000, 0x200020000004c},
+		{0xb12, 0x2000200000000, 0x2000200000584}, // struct pointer
+		{0xb12, 0x2000200000001, 0x2000200000585}, // list pointer
+		{0xb12, 0x2000200000003, 0x2000200000587}, // capability pointer
+	}
+
+	for _, test := range tests {
+		testNearPointer := landingPadNearPointer(test.far, test.tag)
+		if testNearPointer != test.landing {
+			t.Errorf("rawPointer(%#016x).pointerType() = %d; want rawPointer(%#016x).pointerType() = %d", testNearPointer, testNearPointer.pointerType(), test.landing, test.landing.pointerType())
```

Comment:
I've already spent a fair amount of work making rawPointers values easy to debug.  Please rewrite this as:

```go
t.Errorf("landingPadNearPointer(%v, %v) = %v; want %v", test.far, test.tag, landing, test.landing)
```